eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -184,7 +184,7 @@ stages:
                           eng/Update-PkgVersion.ps1 -ServiceDirectory '${{parameters.ServiceDirectory}}' -PackageName '${{artifact.name}}' -PackageDirName '${{artifact.directoryName}}'
                         displayName: Increment package version
                       - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
-                      - template: eng/pipelines/templates/steps/create-pull-request.yml@azure-sdk-tools
+                      - template: ../../../common/pipelines/create-pull-request.yml
```

Comment:
@mitchdenny Do we need to use relative paths like this or can we just do absolute paths from the repo root? `eng/common/pipelines/create-pull-request.yml` I couldn't find any info in the docs on this.

Reply:
The absolute path do not work.